pkg/cmd/roachtest/tests/multitenant_upgrade.go

```diff
@@ -72,6 +72,7 @@ func runMultiTenantUpgrade(ctx context.Context, t test.Test, c cluster.Cluster,
 	kvNodes := c.Node(1)
 
 	settings := install.MakeClusterSettings(install.BinaryOption(predecessorBinary), install.SecureOption(true))
+	settings.Env = append(settings.Env, "COCKROACH_UPGRADE_TO_DEV_VERSION=true")
```

Comment:
lgtm, but we could also append everything in `config.DefaultEnvVars()` here while we're at it

Reply:
I think that's already there?
The line above is `install.MakeClusterSettings` which looks like it consists of `Env:           config.DefaultEnvVars(),`

Reply:
... which means `COCKROACH_UPGRADE_TO_DEV_VERSION` is already in the env. Okay, I have no idea why I am seeing the test pass now but not before. I'm guessing I have a stale binary or something somewhere.

Reply:
yeah i don't see either of these lines referencing `settings`:
```
	tenant11 := createTenantNode(ctx, t, c, kvNodes, tenant11ID, tenantNode, tenant11HTTPPort, tenant11SQLPort, tenantStartOpt)
	tenant11.start(ctx, t, c, predecessorBinary)
```